examples/lib/stories/effects/effects.dart

```diff
@@ -25,9 +26,9 @@ void addEffectsStories(Dashbook dashbook) {
     )
     ..add(
       'Scale Effect',
-      (_) => GameWidget(game: ScaleEffectExample()),
-      codeLink: baseLink('effects/scale_effect_example.dart'),
-      info: ScaleEffectExample.description,
+      (_) => GameWidget(game: OldScaleEffectExample()),
+      codeLink: baseLink('effects/old_scale_effect_example.dart'),
+      info: OldScaleEffectExample.description,
```

Comment:
if v2 for this effect is ready we can just axe it rn? or better axe at the end

Reply:
If we want to release rc17 before the effects are done we need these.